compiler/src/iree/compiler/Dialect/Util/Transforms/CSEInitializers.cpp

```diff
@@ -0,0 +1,137 @@
+// Copyright 2021 The IREE Authors
+//
+// Licensed under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#include "iree/compiler/Dialect/Util/IR/UtilDialect.h"
+#include "iree/compiler/Dialect/Util/IR/UtilOps.h"
+#include "iree/compiler/Dialect/Util/Transforms/Passes.h"
+#include "iree/compiler/Utils/EquivalenceUtils.h"
+#include "llvm/ADT/DenseMap.h"
+#include "llvm/ADT/EquivalenceClasses.h"
+#include "llvm/ADT/SmallVector.h"
+#include "llvm/ADT/StringRef.h"
+#include "llvm/Support/Casting.h"
+#include "mlir/IR/BuiltinAttributes.h"
+#include "mlir/IR/Value.h"
+#include "mlir/Support/LLVM.h"
+#include "mlir/Transforms/Inliner.h"
+#include "mlir/Transforms/InliningUtils.h"
+
+#define DEBUG_TYPE "iree-util-cse-initializer"
+
+namespace mlir::iree_compiler::IREE::Util {
+
+#define GEN_PASS_DEF_CSEINITIALIZERSPASS
+#include "iree/compiler/Dialect/Util/Transforms/Passes.h.inc"
+
+namespace {
+static llvm::cl::opt<bool> clDisableCSEInitializers(
+    "iree-disable-util-cse-initializers",
+    llvm::cl::desc("Disables the iree-util-cse-initializers pass"),
+    llvm::cl::init(false));
+
+bool areAttributesEquivalent(Operation &lhs, Operation &rhs) {
```

Comment:
```suggestion
static bool areAttributesEquivalent(Operation &lhs, Operation &rhs) {
```

always put `static` on functions that are private to a compilation unit (anonymous namespace does the same thing but is less readable - a reader skimming the file has to scroll up and detect if they are in an anonymous region, as code moves it may get moved out of an anonymous region accidentally, etc)

Reply:
This is removed now.